README.md

```diff
@@ -44,15 +45,13 @@ Resources for working with labs and GitHub Actions in general:
 - [Creating a repository from a template]
 - [Viewing workflow run history]
 - [Running a workflow on GitHub]
-- [About billing for GitHub Actions]
 - [GitHub Actions]
 
-<!-- 
-Links used throughout this README: 
+<!--
+Links used throughout this README:
 -->
 [Configuring a publishing source for your GitHub Pages site]:   https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site
 [Creating a repository from a template]:                        https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template
 [Viewing workflow run history]:                                 https://docs.github.com/en/actions/managing-workflow-runs/viewing-workflow-run-history
 [Running a workflow on GitHub]:                                 https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow#running-a-workflow-on-github
-[About billing for GitHub Actions]:                             https://docs.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions
```

Comment:
I do think we need to keep the instructions (and link) to GitHub Actions billing for a few reasons:
- there's a good chance this will be a template that we'll carry over to other labs and learners need to understand the cost implications of taking those labs in private repos
- although we're asking learners to create a public repository, nothing is stopping them from, in fact, creating it privately and incurring charges
- we can't (and don't want to) control how people might fork and use this lab in different ways than the imagined. In the event someone is encouraged to work privately at some point, I'd like us to do our best to ensure they don't get billed accidentally.

Reply:
Ok, good point! I'll re-add this in the README!